docs/cli.md

```diff
@@ -34,39 +34,51 @@ in which it is being run. Currently model analyzer supports 2 modes.
 ### Online Mode
 
 This is the default mode. When in this mode, Model Analyzer will operate to find
-the optimal model configuration for an online inference scenario. By default in 
-online mode, the best model configuration will be the one that maximizes 
-throughput. If a latency budget is specified to the [analyze subcommand](#subcommand-analyze) via 
-`--latency-budget`, then the best model configuration will be the one with the highest throughput in the given budget. 
+the optimal model configuration for an online inference scenario. By default in
+online mode, the best model configuration will be the one that maximizes
+throughput. If a latency budget is specified to the [profile subcommand](#subcommand-profile) via
```

Comment:
do we want to say 'profile or analyze' for cases like this? Or only speak of profile?

Reply:
I only want to speak of profile. With the eventual goal of removing analyze altogether.